app/helpers/config/site_helper.rb

```diff
@@ -159,4 +159,9 @@ def site_document_verification_checklist_url
   def site_invoice_bill_url
     Settings.site.invoice_bill_url
   end
+
+  def check_upcoming_dates?
```

Comment:
You aren't thinking about this correctly.

What if we wanted to support only 30 days? What if we supported 120 days? 

You need relate what we know from this setting value into that behavior